src/admin/containers/payment_reports.js

```diff
@@ -58,12 +60,14 @@ const add_totals = (totals) =>
       ]
     )
   )
-
-const make_data = ({ totals, payments }) =>
-  [ headers
-  , add_totals(totals)(get_fields(payments))
+//TODO format totals numbers using formatPounds function - We are not sure where.
+const make_data = ({ totals, payments }) => {
+  const formatted_payments = map(convert_payment(formatPounds), payments)
```

Comment:
`Table` can be provided with a custom `Row` as a prop, which could be the normal `Row` provided with a custom `Cell`, which could be a `MoneyCell` which divides through by 100